src/components/pages/services/store/actionCreators.tsx

```diff
@@ -0,0 +1,35 @@
+import { ITask, TaskAction, DispatchType } from "../types/type.d";
```

Comment:
Personally, I think it's better to separate all these files under /store/ directory outside the /components/ directory. I think it's better to put them in a src/redux/ directory for example. What do you think?